sql/snapshot.sql

```diff
@@ -0,0 +1,546 @@
+-- public.abstract_blocks definition
+
+-- Drop table
+
+-- DROP TABLE abstract_blocks;
+
+CREATE TABLE abstract_blocks (
+	hash varchar NOT NULL,
+	height int8 NOT NULL,
+	parent jsonb NULL,
```

Comment:
Previously parent had `hash` and `height`. So why jsonb for data that is well structured and won't change dynamically? 🤔